spring-integration-rsocket/src/main/java/org/springframework/integration/rsocket/RSocketConnectedEvent.java

```diff
@@ -67,7 +65,7 @@ public RSocketRequester getRequester() {
 	public String toString() {
 		return "RSocketConnectedEvent{" +
 				"destination='" + this.destination + '\'' +
-				", data=" + Arrays.toString(this.data.asByteBuffer().array()) +
+				", data length=" + this.data.asByteBuffer().array().length +
```

Comment:
This still seems to be a lot of overhead in a `toString()`.

Why do we need this `DataBuffer` in the event?